.gitlab-ci.yml

```diff
@@ -162,16 +162,35 @@ variables:
   # Build images versions
   # To use images from datadog-agent-buildimages dev branches, set the corresponding
   # SUFFIX variable to _test_only
-  DATADOG_AGENT_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_BUILDIMAGES: v32377621-ce42cb8e
-  DATADOG_AGENT_WINBUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_WINBUILDIMAGES: v32377621-ce42cb8e
-  DATADOG_AGENT_ARMBUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_ARMBUILDIMAGES: v32377621-ce42cb8e
-  DATADOG_AGENT_SYSPROBE_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v32377621-ce42cb8e
-  DATADOG_AGENT_BTF_GEN_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_BTF_GEN_BUILDIMAGES: v32377621-ce42cb8e
+  CI_IMAGE_AGENT_DEPLOY_SUFFIX: ""
+  CI_IMAGE_AGENT_DEPLOY_VERSION: v32377621-ce42cb8e
+  CI_IMAGE_BTF_GEN_SUFFIX: ""
+  CI_IMAGE_BTF_GEN_VERSION: v32377621-ce42cb8e
+  CI_IMAGE_DDA_TESTING_SUFFIX: ""
+  CI_IMAGE_DDA_TESTING_VERSION: v32377621-ce42cb8e
+  CI_IMAGE_DEB_ARM64_SUFFIX: ""
+  CI_IMAGE_DEB_ARM64_VERSION: v32377621-ce42cb8e
```

Comment:
You'll need to add an extra tag for the `armhf` image

Reply:
The debian amrhf is built from the very same image definition as the arm64 one, that's why today I believe only one is required

Reply:
It's the same Dockerfile but with different parameters, however in the context of allowing only a partial image rebuild based on the modified files, this would be enough indeed

Reply:
I'd add a comment in the few usages of armhf which mention the arm64 tag though, as it's a bit scary IMO